.env.example

```
@@ -63,6 +72,11 @@ PUBLIC_APP_STORE_EID_WALLET=""
 PUBLIC_PLAY_STORE_EID_WALLET=""
 NOTIFICATION_SHARED_SECRET=your-notification-secret-key
 
+# Shared secret between eid-wallet frontend and evault-core provisioner
+# Backend reads PROVISIONER_SHARED_SECRET, wallet reads PUBLIC_PROVISIONER_SHARED_SECRET
+PROVISIONER_SHARED_SECRET="your-provisioner-shared-secret"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)